charts/external-dns/templates/deployment.yaml

```diff
@@ -126,9 +130,51 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
-      {{- if or .Values.secretConfiguration.enabled .Values.extraVolumes }}
+        {{- if .Values.webhook.image }}
```

Comment:
Wouldn't it be more explicit to rely on Values.provider == "webhook" here?

Reply:
No, .Values.provider is explicitly ignored if a webhook image is specified.